docs/using_eessi/eessi_in_ci.md

```diff
@@ -0,0 +1,49 @@
+# Leveraging EESSI for Continuous Integration
+
+EESSI is already available as both a GitHub Action and a GitLab CI/CD component, which means you can easily integrate
+it if you use continuous integration within those ecosystems.
+
+!!! note
+
+    Both of these EESSI CI tools support the use of [`direnv`](https://direnv.net/) to allow you to store your desired
+    environment within a `.envrc` file within your repository. See the documentation of the individual tools for
+    detailed usage.
+
+## The EESSI GitHub Action
```

Comment:
Would it be overkill to have separate pages for GitHub Actions & GitLab, if only so we can have a nice menu item for both?

Reply:
I can add menu items using the heading anchors if that's desired. I would prefer not to split the page as ultimately I would like the CI to behave the same on all platforms and we can have a definitive place to outline that behaviour.

A good example of why to do it that way is the behaviour of `direnv`. `direnv` works great, you can have any kind of scripting you want in the `.envrc` and (like how Lmod does it) it will run the script and give you the set the environment variables from that environment. However, it _only_ sets environment variables (so no functions), which means if you use it without Lmod initiated then you still can't use Lmod when it is done. I sidestep this right now by initiating the EESSI environment before  using `direnv` and this makes life even easier as `direnv` is already in the our compat layer. Having to explain that multiple times (and keep them in sync) would be a pain.